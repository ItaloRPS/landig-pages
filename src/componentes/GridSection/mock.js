export default {
    background:false,
    title:'My grid',
    description:'Atque doloribud nobis laudantium esse ut,dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore',
    grid:[
        {
            title:'Teste 1',
            description: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!'
            
        },  
        {
            title:'Teste 2',
            description: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!'
            
        },
        {
            title:'Teste 3',
            description: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!'
            
        }

    ]
}