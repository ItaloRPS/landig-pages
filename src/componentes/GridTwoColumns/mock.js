export default {
    title:'Grid Two Columns',
    text: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sit unde debitis vitae nesciunt praesentium ullam nulla saepe deleniti! Recusandae alias, praesentium numquam explicabo rerum esse vel? Explicabo, delectus. Perspiciatis, cupiditate.',
    srcImg: 'assets/imagens/javascript.svg'
}