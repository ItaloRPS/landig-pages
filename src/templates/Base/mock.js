import linksMock from '../../componentes/NavLinks/mock'
import gridMock from '../../componentes/GridSection/mock'
import {GridSection} from '../../componentes/GridSection'

export const mockBase = {
    children:(
        <>
            <GridSection {...gridMock}/>
            <GridSection {...gridMock} background/>
            <GridSection {...gridMock}/>
            <GridSection {...gridMock} background/>
        </>
    ),
        links:linksMock,
        logoData:{
            text:'Logo',
            link:'#'
        },
        footerHtml:'<p> Teste Footer </p>'
    
}
