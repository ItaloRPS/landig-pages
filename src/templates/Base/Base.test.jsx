import {renderTheme} from '../../styles/render-theme'
import {Base} from '.'
import { mocBase } from './mock';

describe('<Base/>',()=>{
    test('should render',()=>{
       const {container}= renderTheme(<Base {...mocBase}/>);
        expect(container).toMatchSnapshot()
    });
});