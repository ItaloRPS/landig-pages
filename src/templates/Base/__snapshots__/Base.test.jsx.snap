// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Base/> should render 1`] = `
.c10 {
  padding-top: 5.4rem;
}

.c4 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

.c6 {
  font-size: 1.6rem;
  text-transform: uppercase;
}

.c13 {
  font-size: 6.4rem;
  text-transform: uppercase;
}

.c18 {
  font-size: 3.2rem;
  text-transform: none;
}

.c20 {
  color: #FFFFFF;
  font-size: 3.2rem;
  text-transform: none;
}

.c2 {
  position: fixed;
  z-index: 5;
  top: 0;
  left: 0;
  right: 0;
  width: 100%;
  background: #FFFFFF;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c2 > .c3 {
  padding-top: 0;
  padding-bottom: 0;
}

.c2 {
  margin-top: 0;
  margin-bottom: 0;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 {
  z-index: 6;
  position: fixed;
  top: 2rem;
  right: 2rem;
  width: 4rem;
  height: 4rem;
  background: #0A1128;
  color: #FFFFFF;
  border: none;
  display: none;
  pointer-events: none;
}

.c0 > svg {
  width: 2.5rem;
  height: 2.5rem;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #0A1128;
  font-size: 2.4rem;
  font: 'Motserrat',sans-serif;
}

.c7 > img {
  max-height: 2.5rem;
}

.c9 {
  display: block;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 1.6rem;
  padding: 2.4rem;
  color: black;
  position: relative;
}

.c9::after {
  content: '';
  position: absolute;
  bottom: 0.75rem;
  left: 50%;
  width: 0%;
  height: 0.2rem;
  background: #dc143c;
}

.c9:hover::after {
  left: 25%;
  width: 50%;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
}

.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c15 {
  font-size: 2.4rem;
}

.c21 {
  text-align: center;
  border-top: 0.1rem solid #DDDDD;
}

.c21 a {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c21 .c14 {
  font-size: 1.6rem;
}

.c21 .c3 {
  padding-top: 0;
  padding-bottom: 0;
}

.c22 {
  position: fixed;
  background-color: #DDDDD;
  color: #FFFFFF;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 4rem;
  height: 4rem;
  bottom: 2rem;
  right: 2rem;
  z-index: 6;
}

.c12 .c14 {
  margin-bottom: 6.4rem;
}

.c16 {
  counter-reset: grid-conunter;
  display: grid;
  grid-template-columns: repeat(auto-fit,minmax(280px,1fr));
  gap: 3.2rem;
}

.c17 {
  position: relative;
  left: 5rem;
}

.c17::before {
  counter-increment: grid-conunter;
  content: counter(grid-conunter);
  position: absolute;
  font-size: 7rem;
  top: -3rem;
  left: -5rem;
  -webkit-transform: rotate(5deg);
  -ms-transform: rotate(5deg);
  transform: rotate(5deg);
  font-weight: 800;
}

.c11 {
  background: #FFFFFF;
  color: #0A1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 5rem;
}

.c19 {
  background: #FFFFFF;
  color: #0A1128;
  background: #0A1128;
  color: #FFFFFF;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 5rem;
}

@media (max-width:768px) {
  .c10 {
    padding-top: 0;
  }
}

@media (max-width:768px) {

}

@media (max-width:768px) {
  .c2 {
    height: 100vh;
    visibility: hidden;
    opacity: 0;
    visibility: visible;
    opacity: 1;
  }

  .c2 > .c3 {
    display: grid;
    grid-template-columns: 1fr;
    grid-template-rows: 1fr;
    height: 100vh;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    overflow-y: auto;
  }

  .c2 {
    padding-bottom: 3.2rem;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

@media (max-width:768px) {
  .c5 {
    display: block;
    text-align: center;
    padding: 4.8rem 0;
  }
}

@media (max-width:768px) {
  .c0 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

@media (max-width:768px) {
  .c8 {
    -webkit-flex-flow: column wrap;
    -ms-flex-flow: column wrap;
    flex-flow: column wrap;
    -webkit-align-content: center;
    -ms-flex-line-pack: center;
    align-content: center;
  }
}

@media (max-width:768px) {
  .c16 {
    grid-template-columns: 1fr;
  }
}

<div>
  <button
    aria-label="Open/Close Menu"
    class="c0"
  >
    <svg
      aria-hidden="true"
      aria-label="Close Menu"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"
      />
    </svg>
  </button>
  <div
    class="c2"
  >
    <div
      class="c3 c4"
    >
      <div
        class="c5"
      >
        <h1
          class="c6"
        >
          <a
            class="c7"
            href="#"
          >
            <span>
              Logo
            </span>
          </a>
        </h1>
        <nav
          class="c8"
        >
          <a
            class="c9"
            href="#target1"
            target="_self"
          >
            Link1
          </a>
          <a
            class="c9"
            href="#target2"
            target="_self"
          >
            Link2
          </a>
          <a
            class="c9"
            href="#target3"
            target="_self"
          >
            Link3
          </a>
          <a
            class="c9"
            href="#target4"
            target="_self"
          >
            Link4
          </a>
          <a
            class="c9"
            href="#target5"
            target="_self"
          >
            Link5
          </a>
          <a
            class="c9"
            href="#target6"
            target="_self"
          >
            Link6
          </a>
          <a
            class="c9"
            href="#target7"
            target="_self"
          >
            Link7
          </a>
          <a
            class="c9"
            href="#target8"
            target="_self"
          >
            Link8
          </a>
        </nav>
      </div>
    </div>
  </div>
  <div
    class="c10"
  >
    <div
      class="c11"
    >
      <div
        class="c12"
      >
        <h1
          class="c13"
        >
          My grid
        </h1>
        <div
          class="c14 c15"
        >
          Atque doloribud nobis laudantium esse ut,dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore
        </div>
        <div
          class="c16"
        >
          <div
            class="c17"
          >
            <h3
              class="c18"
            >
              Teste 1
            </h3>
            <div
              class="c14 c15"
            >
              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!
            </div>
          </div>
          <div
            class="c17"
          >
            <h3
              class="c18"
            >
              Teste 2
            </h3>
            <div
              class="c14 c15"
            >
              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!
            </div>
          </div>
          <div
            class="c17"
          >
            <h3
              class="c18"
            >
              Teste 3
            </h3>
            <div
              class="c14 c15"
            >
              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c19"
    >
      <div
        class="c12"
      >
        <h1
          class="c13"
        >
          My grid
        </h1>
        <div
          class="c14 c15"
        >
          Atque doloribud nobis laudantium esse ut,dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore
        </div>
        <div
          class="c16"
        >
          <div
            class="c17"
          >
            <h3
              class="c20"
            >
              Teste 1
            </h3>
            <div
              class="c14 c15"
            >
              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!
            </div>
          </div>
          <div
            class="c17"
          >
            <h3
              class="c20"
            >
              Teste 2
            </h3>
            <div
              class="c14 c15"
            >
              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!
            </div>
          </div>
          <div
            class="c17"
          >
            <h3
              class="c20"
            >
              Teste 3
            </h3>
            <div
              class="c14 c15"
            >
              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c11"
    >
      <div
        class="c12"
      >
        <h1
          class="c13"
        >
          My grid
        </h1>
        <div
          class="c14 c15"
        >
          Atque doloribud nobis laudantium esse ut,dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore
        </div>
        <div
          class="c16"
        >
          <div
            class="c17"
          >
            <h3
              class="c18"
            >
              Teste 1
            </h3>
            <div
              class="c14 c15"
            >
              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!
            </div>
          </div>
          <div
            class="c17"
          >
            <h3
              class="c18"
            >
              Teste 2
            </h3>
            <div
              class="c14 c15"
            >
              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!
            </div>
          </div>
          <div
            class="c17"
          >
            <h3
              class="c18"
            >
              Teste 3
            </h3>
            <div
              class="c14 c15"
            >
              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c19"
    >
      <div
        class="c12"
      >
        <h1
          class="c13"
        >
          My grid
        </h1>
        <div
          class="c14 c15"
        >
          Atque doloribud nobis laudantium esse ut,dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore
        </div>
        <div
          class="c16"
        >
          <div
            class="c17"
          >
            <h3
              class="c20"
            >
              Teste 1
            </h3>
            <div
              class="c14 c15"
            >
              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!
            </div>
          </div>
          <div
            class="c17"
          >
            <h3
              class="c20"
            >
              Teste 2
            </h3>
            <div
              class="c14 c15"
            >
              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!
            </div>
          </div>
          <div
            class="c17"
          >
            <h3
              class="c20"
            >
              Teste 3
            </h3>
            <div
              class="c14 c15"
            >
              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c21"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c14 c15"
        >
          <p>
             Teste Footer 
          </p>
        </div>
      </div>
    </div>
  </div>
  <a
    aria-label="Go to top"
    class="c22"
    href="#"
    title="Go to top"
  >
    <svg
      aria-hidden="true"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
      />
    </svg>
  </a>
</div>
`;
